### ISSUE TITLE
Stop querying pypi when OUTDATED_IGNORE is on

### ISSUE THREAD

**Issue by 0x26res** [alexmojaki/outdated#13]
My understanding of how outdated works is that if the environment variable `OUTDATED_IGNORE` is set (to any non empty value), then it will stop warning when a package is outdated.

But this will still query pypi in the background. Again, my understanding is that this will happen in a different thread and should not have a detrimental impact on performance.

Nevertheless, some people don't want to have their production environment attempt to query pypi, for various reason (security being the main one).

Would it be possible to change the behaviour of OUTDATED_IGNORE so that it makes `outdated` do absolutely nothing. or alternatively add another environment variable to disable checks altogether.

Thanks.

**Comment by alexmojaki** [comment]
Good point, I don't know why I didn't do this in the first place. I'd be happy to accept a PR which makes OUTDATED_IGNORE prevent anything from happening at all.

**Comment by 0x26res** [comment]
OK, I'll give it a try, thanks.
